gp-includes/formats/format_pomo.php

```diff
@@ -31,11 +32,36 @@ public function print_exported_file( $project, $locale, $translation_set, $entri
 		foreach( $entries as $entry ) {
 			$po->add_entry( $entry );
 		}
-		$po->set_header( 'Project-Id-Version', $project->name );
 
-		// TODO: include parent project's names in the comment
-		$po->comments_before_headers .= "Translation of {$project->name} in {$locale->english_name}\n";
-		$po->comments_before_headers .= "This file is distributed under the same license as the {$project->name} package.\n";
+		$current = $project;
+		$project_tree = array();
+		$project_tree[] = $current->name;
+
+		while ( $current->parent_project_id > 0 ) {
+			$current = GP::$project->get( $project->parent_project_id );
+			$project_tree[] = $current->name;
+		}
```

Comment:
I think this should be part of `GP_Project`. Maybe `GP_Project::get_full_name( $separator = ' - ' ) { }`